src/walletbackend/Transfer.cpp

```diff
@@ -26,15 +27,18 @@ namespace SendTransaction
            as the static constructors were used */
         const std::string defaultAddress = subWallets->getPrimaryAddress();
 
-        return sendFusionTransactionAdvanced(defaultMixin, {}, defaultAddress, daemon, subWallets);
+        const std::vector<uint8_t> extraData;
+
+        return sendFusionTransactionAdvanced(defaultMixin, {}, defaultAddress, daemon, subWallets, extraData);
```

Comment:
If we want to be terser here, we could just do 
```suggestion
        return sendFusionTransactionAdvanced(defaultMixin, {}, defaultAddress, daemon, subWallets, {});
```

Might complain if we're taking a reference, though.

Reply:
Worked for both the sendFusionTransactionBasic and sendTransactionBasic. Updated.